### PR TITLE
[WIP] Fixed signal not sending multiple messages.

### PR DESCRIPTION
**Pull request by Elland**

### CODE REVIEW

File: Toshi/Controllers/Messaging/ChatInteractor.swift [hunk 1]
```diff
@@ -377,16 +389,16 @@ final class ChatsInteractor {
...
                     if let thumbnailData = mediaData?["thumbnailData"] as? Data {
-                        let outgoingMessage = TSOutgoingMessage(timestamp: timestamp, in: self.thread, messageBody: wrapper.content)
-                        self.messageSender?.sendAttachmentData(thumbnailData, contentType: contentType, sourceFilename: "File.jpeg", in: outgoingMessage, success: {
-                            print("Success")
-                        }, failure: { error in
-                            print("Failure: \(error)")
-                        })
+                        let fileExtension = contentType.components(separatedBy: "/").last ?? ""
+                        let name = (mediaData!["assetIdentifier"] as? String ?? "").replacingOccurrences(of: "/", with: "-")
+                        let filename = name.appending(".\(fileExtension)")
```
Comment (yuliaveres) [inline Toshi/Controllers/Messaging/ChatInteractor.swift]: great

File: Toshi/Controllers/Messaging/ChatInteractor.swift [hunk 2]
```diff
@@ -377,16 +389,16 @@ final class ChatsInteractor {
...
+
+                        // By delaying each message by a little bit, we avoid an issue
+                        // where only the first attachment would be sent.
+                        DispatchQueue.main.asyncAfter(seconds: 0.1) {
```
Comment (yuliaveres) [inline Toshi/Controllers/Messaging/ChatInteractor.swift]: should it be on the main ?

Comment (yuliaveres) [inline Toshi/Controllers/Messaging/ChatInteractor.swift]: Also not sure if the delay helps, i still do receive only 1 image on other part

File: Toshi/Controllers/Messaging/ChatController.swift [hunk 3]
```diff
@@ -885,6 +885,7 @@ extension ChatController: ImagePickerDelegate {
     }
     
     func wrapperDidPress(_: ImagePickerController, images _: [UIImage]) {
+        print("!")
```
Comment (yuliaveres) [inline Toshi/Controllers/Messaging/ChatController.swift]: nice

**Comment by Elland** [comment]
No longer relevant.

**Pull request closed without merging**